src/pages/naraAn/Main/Main.js

```diff
@@ -1,7 +1,73 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 
-function MainNara() {
-  return <h1>naraMain</h1>;
+import RecommendUser from './components/RecommendUser';
+import AsideFooter from './components/AsideFooter';
+import Nav from '../../../components/Nav/Nav';
+import StoryUser from './components/StoryUser';
+import Feeds from './Feeds';
+import './Main.scss';
+
+function Main() {
+  const [feedInfoList, setFeedInfoList] = useState([]);
+  const userWecode = 'wecode_37th';
+
+  useEffect(() => {
+    fetch('../../../../data/feedData.json')
```

Comment:
```suggestion
    fetch('/data/feedData.json')
```
- public에 접근하는 경로는 이렇게도 접근 가능합니다.